pages/partials/question.ejs

```diff
@@ -38,6 +38,17 @@
           Show/hide
         </button>
       </p>
+      <p><strong> Issue status: </strong>
+        <% if (issue.open) { %>
+        <span class="badge badge-danger" data-toggle="collapse" href="#">
```

Comment:
Can we use the same icon styling as the issue badges elsewhere? https://github.com/PrairieLearn/PrairieLearn/blob/master/pages/instructorIssues/instructorIssues.ejs#L168

That is, add the `fa-exclamation-circle` and `fa-check-circle` icons.